docs/package.json

```diff
@@ -26,11 +26,11 @@
     "@emotion/react": "^11.10.4",
     "@emotion/server": "^11.10.0",
     "@emotion/styled": "^11.10.4",
-    "@mui/icons-material": "^5.10.3",
-    "@mui/joy": "^5.0.0-alpha.45",
-    "@mui/material": "^5.10.5",
-    "@mui/styles": "^5.10.3",
-    "@mui/utils": "^5.10.3",
+    "@mui/icons-material": "^5.11.11",
+    "@mui/joy": "^5.0.0-alpha.69",
```

Comment:
The out-of-sync joy version was the main culprit of TSC issues, but it made sense to sync/update all relevant MUI packages. 🤔